pkg/config/config.go

```diff
@@ -136,35 +158,35 @@ func ParseConfig(path string) error {
 }
 
 func ParseConfigBytes(data []byte) error {
-	err := json.Unmarshal(data, &config)
+	err := json.Unmarshal(data, &defaultConfig)
```

Comment:
hm, now I see why it was called `config` and not `defaultconfig` it gets confusing when we use it here. Can we change this to make it more explicit that the default confit is parsed into the final config to use. Something like:
```suggestion
	config := &defaultConfig;
	err := json.Unmarshal(data, config)
```
This doesn't change anything, but later it is more clear when we have something like:`config.PublicAddress` , instead of `defaultConfig.PublicAddress` - this one looks like we still use the default config which is misleading.